.github/renovate.json5

```
@@ -0,0 +1,52 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval",
```

`:dependencyDashboardApproval` [^1] means you'll have to approve _all_ PRs from the Dependency Dashboard. Unless you set excemptions in `packageRules` below... but that is extra work. 😄 

I think you actually want `:approveMajorUpdates` [^2], where you only approve `major` updates beforehand, and get all other updates automatically?

```suggestion
        ":approveMajorUpdates",
```

[^1]: [Renovate docs, `:approveMajorUpdates` preset](https://docs.renovatebot.com/presets-default/#approvemajorupdates)
[^2]: [Renovate docs, `:dependencyDashboardApproval`](https://docs.renovatebot.com/presets-default/#dependencydashboardapproval)